Library/Power Ups/Power Ups 8 Limitations Limitation Modifiers.adm

```diff
@@ -0,0 +1,790 @@
+		<affects>total</affects>
+		<notes>@Disadvantage@</notes>
+	</modifier>
+	<modifier version="1">
+		<name>Aftermath, 2 FP/Level</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-5</cost>
+		<levels>1</levels>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>All-Out</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-25</cost>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>All-Out, Concentrate</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-25</cost>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>Aspected (@Aspect@)</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-20</cost>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>Attracts Threats</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-5</cost>
+		<affects>total</affects>
+		<notes>CR 6</notes>
+	</modifier>
+	<modifier version="1">
+		<name>Attracts Threats</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-10</cost>
+		<affects>total</affects>
+		<notes>CR 9</notes>
+	</modifier>
+	<modifier version="1">
+		<name>Backlash (@Attribute Penalty, Incapacitation, Irritant, Stunning@)</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-1</cost>
+		<levels>1</levels>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>Backlash, Resistable (@Attribute Penalty, Incapacitation, Irritant, Stunning@)</name>
+		<reference>PU8:11</reference>
+		<cost type="percentage">-1</cost>
+		<levels>1</levels>
+		<affects>total</affects>
+		<notes>Half value of Unresistable</notes>
+	</modifier>
+	<modifier version="1">
+		<name>Blockable</name>
+		<reference>PU8:12</reference>
+		<cost type="percentage">-5</cost>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>Can Be Parried</name>
+		<reference>PU8:12</reference>
+		<cost type="percentage">-5</cost>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>Blockable Or Parryable</name>
+		<reference>PU8:12</reference>
+		<cost type="percentage">-10</cost>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>Blood Agent, Reversed</name>
+		<reference>PU8:12</reference>
+		<cost type="percentage">-40</cost>
+		<affects>total</affects>
+	</modifier>
+	<modifier version="1">
+		<name>Cardiac Stress</name>
+		<reference>PU8:12</reference>
+		<cost type="percentage">-50</cost>
+		<affects>total</affects>
+		<notes>Every second</notes>
+	</modifier>
+	<modifier version="1">
+		<name>Cardiac Stress</name>
+		<reference>PU8:12</reference>
+		<cost type="percentage">-40</cost>
+		<affects>total</affects>
+		<notes>Every 10 seconds</notes>
+	</modifier>
+	<modifier version="1">
+		<name>Cardiac Stress</name>
+		<reference>PU8:12</reference>
+		<cost type="percentage">-30</cost>
+		<affects>total</affects>
+		<notes>Every minute</notes>
```

Comment:
I really need to have a way to provide a formula or table lookup for doing these types of things. It's a shame that you have to repeat essentially the same thing with a small text and point cost change over and over.